### ISSUE TITLE
How to train the model to achieve consistent results?

### ISSUE THREAD

**Issue by caijunhao** [TheCamusean/grasp_diffusion#10]
Hi [@TheCamusean](https://github.com/TheCamusean),

Thanks for your great work! 
I'm trying to train a point cloud-conditioned model from scratch based on the parameters provided in the code. However, the evaluation results seem to have a large difference from those evaluated with the checkpoints you provided (44% vs. 92% of grasp success rates). 
Is there any hyperparameter I need to modify? If not, what should I do to achieve consistent results?
Thank you!

**Comment by robotgradient** [comment]
Training on Pointclouds usually takes a long time. 

Even if you observe the learning rate of the diffusion getting constant, it is good to keep training the model. 

For how long did you train your model?

Best regards,
-Julen

**Comment by caijunhao** [comment]
Thank you for your reply. 
I tried to increase the batch size to 16 and trained the model for about a day, and now it can achieve the same results.
